base/string_utils.cpp

```diff
@@ -220,6 +220,10 @@ bool IsASCIIString(string const & str)
   return true;
 }
 
+bool IsASCIIDigit(UniChar c) { return c >= '0' && c <= '9'; }
+
+bool IsASCIILatin(UniChar c) { return (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z'); }
```

Comment:
Там видимо некоторые ведра были с нестандартной локалью?

> On 22.4.2016, at 18.06, Viktor Govako <notifications@github.com> wrote:
> 
> In base/string_utils.cpp:
> 
> > @@ -220,6 +220,10 @@ bool IsASCIIString(string const & str)
> >    return true;
> >  }
> >  
> > +bool IsASCIIDigit(UniChar c) { return c >= '0' && c <= '9'; }
> > +
> > +bool IsASCIILatin(UniChar c) { return (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z'); }
> isdigit когда-то бажил на ведре. См feature_impl.cpp
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>